composer.json

```
@@ -26,7 +26,7 @@
         }
     ],
     "require":{
-        "php": ">=7.4",
+        "php": "^7.4 || ^8.0",
```

```suggestion
        "php": ">=7.4",
```

I want to leave the PHP requirement in my extensions open ended 🙃